### DISCUSSION TITLE
How to install all librairies from a particular renv.lock file ?

### DISCUSSION THREAD

**Discussion by ThomasFaria** [InseeFrLab/ESA-Nowcasting-2023#57] [Q&A] [answered]
How to install all librairies from a particular renv.lock file ?

**Comment by antoine-palazz** [comment, marked as answer]
You can use the ```renv::restore()``` function to install all the libraries listed in the renv.lock file.
This will install all the packages with their respective versions listed in the renv.lock file, ensuring reproducibility of the R environment.

Note that the renv.lock file should be present in the project directory and should contain a snapshot of the library versions used in the project. If the renv.lock file is not present, you can create it by running ```renv::snapshot()```.
